sdk/tbrv3/solana/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "types": [
+      "mocha",
+      "chai"
+    ],
+    "lib": [
+      "ES2023"
+    ],
+    "module": "CommonJS",
+    "moduleResolution": "Node",
+    "target": "es6",
+    "resolveJsonModule": true,
+    "esModuleInterop": true
+  },
+  "include": [
+    "src",
+    "../../../target/idl",
+    "index.ts"
```

Comment:
I think this should be included in the `files` property instead.

Reply:
Done